### DISCUSSION TITLE
Bug after latest update within snacks

### DISCUSSION THREAD

**Discussion by SargisPlusPlus** [LazyVim/LazyVim#4863] [Q&A]
![Screenshot 2024-11-21 at 10 50 19 AM](https://github.com/user-attachments/assets/260ff4a6-7e07-41ca-8b87-33cd0254e3a2)

**Comment by smackysnacks** [comment]
Looks like it was reported https://github.com/folke/snacks.nvim/issues/135

**Reply by folke** [reply]
fixed

**Reply by SargisPlusPlus** [reply]
@folke, as usual, thanks for such quick turn around.
